### PR TITLE
SASS-8744 - UK Rent a Room:'About' section - Implement Welsh Translation

### PR DESCRIPTION
**Pull request by modivilioglu**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -1240,3 +1240,88 @@ rentalsAndRentARoomIncomeStart.l3 = *Missing Welsh*
...
+claimExpensesOrRRR.amount.error.twoDecimalPlaces.agent = Gall cyfanswm y Rhyddhad Rhentu Ystafell a hawliwyd gynnwys punnoedd a cheiniogau yn unig, er enghraifft ¬£600 neu ¬£600.20
+claimExpensesOrRRR.amount.error.outOfRange = Nodwch swm rhwng ¬£{0} a ¬£{1}
+claimExpensesOrRRR.change.hidden = Hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.rentARoom = Rhyddhad Rhentu Ystafell
+claimExpensesOrRRR.expenses = Treuliau
+claimExpensesOrRRR.radio.question.label.individual = Faint yr hoffech ei hawlio?
+claimExpensesOrRRR.radio.question.label.agent = Faint hoffai’ch cleient ei hawlio?
+claimExpensesOrRRR.para1.jointlyLet.individual = Gallwch hawlio hyd at ¬£3,750 y flwyddyn yn rhydd o dreth drwy rentu ystafell yn eich cartref. Os byddwch yn hawlio’r rhyddhad hwn, ni fyddwch yn gallu hawlio treuliau ar gyfer yr un ystafell.
+claimExpensesOrRRR.para1.notJointlyLet.individual = Gallwch hawlio hyd at ¬£7,500 y flwyddyn yn rhydd o dreth drwy rentu ystafell yn eich cartref. Os byddwch yn hawlio’r rhyddhad hwn, ni fyddwch yn gallu hawlio treuliau ar gyfer yr un ystafell.
```
Comment (arunkumar461) [inline conf/messages.cy:1325]: Content changed in English, pls compare English text and the one we requested. Pls test all screens

Refer this commit: f1c3d32324b6a2f1b834b8b12047533aacfd2c48

**Review by arunkumar461 — CHANGES_REQUESTED** [review]

File: conf/messages.cy (hunk 1 above)
Comment (modivilioglu) [reply conf/messages.cy]: I've removed the unused messages, and parameterised the first two ones on the Welsh translation. Regarding testing all screens, I believe it is QA's responsibility, however, sure I'll do a short check to make sure the Welsh translation is working.

Comment (arunkumar461) [reply conf/messages.cy]: thanks @modivilioglu for addressing the issue, when I mean all screens, the files you have changed part of this PR

File: conf/messages.cy [hunk 2]
```diff
@@ -1240,3 +1240,86 @@ rentalsAndRentARoomIncomeStart.l3 = *Missing Welsh*
...
+claimExpensesOrRRR.amount.error.twoDecimalPlaces.individual = Gall cyfanswm y Rhyddhad Rhentu Ystafell a hawliwyd gynnwys punnoedd a cheiniogau yn unig, er enghraifft ¬£600 neu ¬£600.20
+claimExpensesOrRRR.amount.error.twoDecimalPlaces.agent = Gall cyfanswm y Rhyddhad Rhentu Ystafell a hawliwyd gynnwys punnoedd a cheiniogau yn unig, er enghraifft ¬£600 neu ¬£600.20
+claimExpensesOrRRR.amount.error.outOfRange = Nodwch swm rhwng ¬£{0} a ¬£{1}
+claimExpensesOrRRR.change.hidden = Hawlio treuliau neu Ryddhad Rhentu Ystafell
+claimExpensesOrRRR.rentARoom = Rhyddhad Rhentu Ystafell
+claimExpensesOrRRR.expenses = Treuliau
+claimExpensesOrRRR.radio.question.label.individual = Faint yr hoffech ei hawlio?
+claimExpensesOrRRR.radio.question.label.agent = Faint hoffai’ch cleient ei hawlio?
+claimExpensesOrRRR.para1.jointlyLet.individual = Gallwch hawlio hyd at £{0} y flwyddyn yn rhydd o dreth drwy rentu ystafell yn eich cartref. Os byddwch yn hawlio’r rhyddhad hwn, ni fyddwch yn gallu hawlio treuliau ar gyfer yr un ystafell.
```
Comment (arunkumar461) [inline conf/messages.cy:1324]: `claimExpensesOrRRR.para1.jointlyLet.agent ` ? Pls test again the files you have changed

**Review by arunkumar461 — CHANGES_REQUESTED** [review]

**Review by arunkumar461 — APPROVED** [review]

**Pull request merged**